internal/service/project/model_project_test.go

```diff
@@ -122,32 +185,33 @@ func TestLimitsDataSourceSDKToTFModel(t *testing.T) {
 	for _, tc := range testCases {
 		t.Run(tc.name, func(t *testing.T) {
 			resultModel := project.NewTFLimitsDataSourceModel(context.Background(), tc.dataFederationLimits)
-			if !reflect.DeepEqual(resultModel, tc.expectedTFModel) {
+			if !assert.Equal(t, tc.expectedTFModel, resultModel) {
```

Comment:
assert.NotEqual

Reply:
actually we want to run assert.Equal, the `if` is to add a message if it fails. I think we can remove the if and additional message all together since the assert method provides all the details when test fails.

Reply:
you're right, not if require

Reply:
adjusted